plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/config/kafka/OpenLineageClientKafkaTransportConfig.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.openlineage.config.kafka;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import jakarta.validation.constraints.NotNull;
+
+public class OpenLineageClientKafkaTransportConfig
+{
+    private String topicName;
+    private String messageKey;
+    private String brokerEndpoints;
+
+    @NotNull
+    public String getTopicName()
+    {
+        return topicName;
+    }
+
+    @Config("openlineage-event-listener.transport.kafka.topic")
+    @ConfigDescription("String specifying the topic to which events will be sent")
+    public OpenLineageClientKafkaTransportConfig setTopicName(String topicName)
```

Comment:
if the config is `openlineage-event-listener.transport.kafka.topic` can we unify this and use `topic` here instead of `topicName`?